console/src/components/Services/Data/TableModify/ComputedFieldsEditor.tsx

```diff
@@ -62,27 +77,28 @@ const ComputedFieldsEditor = ({
     let origComputedFieldFunctionName = '';
     let origComputedFieldComment = '';
     if (origComputedField) {
-      const origComputedFieldFunctionDef = getQualifiedTableDef(
-        origComputedField.definition.function
-      );
+      const origComputedFieldFunctionDef =
+        origComputedField.definition.function;
 
       origComputedFieldName = origComputedField.computed_field_name;
       origComputedFieldFunctionName = origComputedFieldFunctionDef.name;
-      origComputedFieldComment = origComputedField.comment;
+      if (origComputedField?.comment) {
+        origComputedFieldComment = origComputedField.comment;
+      }
```

Comment:
A user might have removed a comment. In this case, `origComputedField.comment` would be falsy. As if condition would be `false` for it, then we won't be able to remove a comment.